sdk/scheduler/src/test/java/com/mesosphere/sdk/api/PlansResourceTest.java

```diff
@@ -15,18 +19,24 @@
 import java.util.Collections;
 import java.util.UUID;
 
+import static com.mesosphere.sdk.api.ResponseUtils.alreadyReportedResponse;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.*;
-import static com.mesosphere.sdk.api.ResponseUtils.*;
 
 public class PlansResourceTest {
-    @Mock private Plan mockPlan;
-    @Mock private Strategy<Phase> mockPlanStrategy;
-    @Mock private Phase mockPhase;
-    @Mock private Strategy<Step> mockPhaseStrategy;
-    @Mock private Step mockStep;
-    @Mock private PlanScheduler planScheduler;
+    @Mock
```

Comment:
I was just using @benclarkwood's IntelliJ settings. Not sure what he was using for this. I have reverted so as to decrease the noise in this PR.